lib/core/logconf.c

```diff
@@ -498,7 +498,7 @@ char *h2o_log_request(h2o_logconf_t *logconf, h2o_req_t *req, size_t *len, char
             break;
         case ELEMENT_TYPE_ENV_VAR:  /* %{..}e */  {
             h2o_iovec_t *env_var = h2o_req_getenv(req, element->data.name.base, element->data.name.len, 0);
-            if (env_var == NULL || env_var->len < 1)
+            if (env_var == NULL || env_var->len < 0)
```

Comment:
Thank you for the change. Actually, you do not need to do `env_var->len < 0`, since a value of `size_t` is guaranteed to be non-negative.